pkg/controller/direct/managedkafka/consumergroup_mappings.go

```diff
@@ -12,7 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +tool:fuzz-gen
```

Comment:
Let's remove the remaining lines in this annotation block as well. They’re only relevant when the file is generated.